tests/test_config.py

```diff
@@ -586,6 +586,30 @@ def test_bool(
             sd = ScriptDirectory.from_config(cfg)
             eq_(getattr(sd, paramname), bool(argtype.true))
 
+    def test_toml_int_value(self, pyproject_only_env):
+        cfg = pyproject_only_env
+        with cfg._toml_file_path.open("wb") as file_:
+            file_.write(
+                rb"""
+[tool.alembic]
+script_location = "%(here)s/scripts"
+
+my_int = 42
+my_int_zero = 0
+my_int_negative = -7
+my_int_large = 12345678901234567890
+my_int_str = "123"
+"""
+            )
+        if "toml_alembic_config" in cfg.__dict__:
+            cfg.__dict__.pop("toml_alembic_config")
+
+        eq_(cfg.get_alembic_option("my_int"), "42")
```

Comment:
the variety of int values look to be testing more tomllib than alembic, but IMO these should be ints.

bigger issue is I think there needs to be a test for the actual truncate_slug_length parameter being present in the config.   do a `sd = ScriptDirectory.from_config()` then `eq_(sd.truncate_slug_length, <value>)`

Reply:
Could you take another look? :)